robotpy_ext/autonomous/stateful_autonomous.py

```diff
@@ -218,7 +218,9 @@ def __init__(self, components=None):
             for k,v in components.items():
                 setattr(self, k, v)
         
-        self.__table = networktables.NetworkTable.getTable('SmartDashboard')
+        # self.__table = networktables.NetworkTable.getTable('SmartDashboard')
+        self.__table = networktables.NetworkTable.getTable('autonomous/%s/stateful' % getattr(self, 'MODE_NAME'  ))
```

Comment:
Because then the metadata like descriptors and durations have their own separation. Although if we got rid of the arrays, it may be better to put it in the same table as the 'tunables'